src/main/java/wedlog/address/model/person/DietaryRequirements.java

```diff
@@ -65,6 +65,11 @@ public boolean equals(Object other) {
             return true;
         }
 
+        // Dietary Requirement where Status.NULL is equal to null object
+        if (other == null) {
+            return this.status.equals(Status.NULL);
+        }
+
```

Comment:
Personally, I don't think this should be true. Do you have a particular use case in mind in writing this line?